hxinsight-extension/src/main/resources/alfresco/module/alfresco-hxinsight-connector-hxinsight-extension/alfresco-global.properties

```diff
@@ -17,3 +17,5 @@ hxi.auth.retry.initial-delay=500
 hxi.auth.retry.delay-multiplier=2
 
 hxi.client.baseUrl=http://localhost:8001
+
+hxi.question.max-nodes-to-ask-about=100
```

Comment:
I like everything about this PR except the name of the option. Maybe `max-nodes-in-question-context` or `max-context-size-for-question`?